homeassistant/components/volkszaehler/__init__.py

```diff
@@ -1 +1,128 @@
 """The volkszaehler component."""
+
+from datetime import timedelta
+import logging
+import re
+
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PORT
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.typing import ConfigType
+
+from .const import (
+    CONF_FROM,
+    CONF_MONITORED_CONDITIONS,
+    CONF_SCANINTERVAL,
+    CONF_TO,
+    CONF_UUID,
+    DEFAULT_HOST,
+    DEFAULT_MONITORED_CONDITIONS,
+    DEFAULT_NAME,
+    DEFAULT_PORT,
+    DEFAULT_SCANINTERVAL,
+    DOMAIN,
+)
+
+CONFIG_SCHEMA = cv.platform_only_config_schema(DOMAIN)
```

Comment:
Please start the import via the existing YAML entrypoint (`sensor.py` -> `async_setup_platform`)